pkg/service/backup/worker_schema.go

```diff
@@ -72,6 +72,13 @@ func (w *worker) safeBackupAndRestoreSchemaDump(ctx context.Context, descSchemaH
 	return nil
 }
 
+func (w *worker) raftReadBarier(ctx context.Context, session gocqlx.Session, useReadBarierAPI bool) error {
+	if useReadBarierAPI {
+		return w.Client.RaftReadBarier(ctx, "", 0)
```

Comment:
Setting timeout to 0 results in setting it to 60s (taken from swagger docs).
On the other hand, scyllaclient has it's own timeout configuration.
I would say that it's better to align those timeouts (retries can probably stay the same).

You can use `customTimeout` for this.

Reply:
yeah, ok, I'll align them

BTW: where I can find config files with scyllaclient timeout defined?